src/promptflow/tests/test_configs/eager_flows/basic_callable_class/simple_callable_class.py

```diff
@@ -17,13 +19,14 @@ def __init__(self, obj_input: str):
 
     @trace
     def __call__(self, func_input: str) -> FlowOutput:
-        return {
-            "obj_input": self.obj_input,
-            "func_input": func_input,
-            "obj_id": id(self),
-        }
+        return FlowOutput(obj_input=self.obj_input, func_input=func_input, obj_id=id(self))
 
     def __aggregate__(self, results: list) -> dict:
+        # Try attribute-style access for the datacalss
+        obj_inputs = [r.obj_input for r in results]
+        func_inputs = [r.func_input for r in results]
+        obj_ids = [r.obj_id for r in results]
+
         return {"length": len(results)}
```

Comment:
Do we cover the scenario that return result is a raw string?

Reply:
Added.